### ISSUE TITLE
Custom action button

### ISSUE THREAD

**Issue by Jamalianpour** [playmoweb/markdown-editable-textinput#33]
Hello,
Can we add some custom action button to list of actions?

**Comment by MrAntonS** [comment]
I agree, I'm trying to implement an action to get image from the device, upload it to the server, and insert url into image markdown
